tests/sentry/snuba/metrics/test_metrics_layer/test_metrics_enhanced_performance.py

```diff
@@ -240,6 +240,61 @@ def test_alias_on_same_metrics_expression_but_different_aliases(self):
             key=lambda elem: elem["name"],
         )
 
+    def test_custom_measurement_query_with_valid_mri(self):
```

Comment:
Please add a test for `d:sessions/measurements.speed@millisecond`, and that should fail 
and also add a test for `s:transactions/measurements.speed@millisecond` and should fail too

Reply:
Also make sure to freeze time so we don't end up with flaky tests again